### ISSUE TITLE
There's any way that I can change my flatpak app icon?

### ISSUE THREAD

**Issue by CarlosSMA** [flatpak/flatpak#1838]
I'm trying to change an icon from a game called "Dead Cells", actually, I remember that (when I didn't format my pc), i've made it, but now I've made all the steps and nothing.

What I did was
1- Download the icon from google
2- Rename it as 'org.flatpaklinux.DeadCells.png'
3- Switch it for the folder '/var/łib/flatpak/exports/share/icons/hicolor/'.

I already switched the icon folder in the .desktop file, and also did the same process but in the .local/ folder, and nothing. Any idea?

**Comment by TingPing** [comment]
Putting it in `~/.local/share/icons/hicolor/$SIZE/apps/$APPID.png` and running `gtk-update-icon-cache -ft ~/.local/share/icons/hicolor` should be the correct solution.

**Comment by Pshemas** [comment]
I'm using Oranchello theme. It does not have png files stored on "per-resolution folder" basis. It has "scalable" folder with SVGs. So where should I put those?
I've tried a couple of locations. I even exported those to PNG, but to use an example I can't get it to work with Krita. Tried using naming it as krita.png , org.kde.krita.png - it does not work.
ANy suggestions how can I make custom icons for Flatpak apps?
